easyfinance.client/src/styles.css

```diff
@@ -47,3 +47,6 @@ body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
 .btn-default:hover {
   background-color: #f8f9fa;
 }
+
+html, body { height: 100%; }
+body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
```

Comment:
This part of the code already exists in the same archive in the beginning.

```suggestion
```

Reply:
It was added automatically by Angular material. I removed it now.